src/core/components/layout-utils.jsx

```diff
@@ -186,7 +186,7 @@ export class Select extends React.Component {
 
   render(){
     let { allowedValues, multiple, allowEmptyValue, disabled } = this.props
-    let value = this.state.value.toJS ? this.state.value.toJS() : this.state.value
+    let value = this.state.value?.toJS?.() || this.state.value
```

Comment:
New syntax: [Optional chaining](https://github.com/tc39/proposal-optional-chaining)!

Stage 2 is usually a little early to be buying in, but given the history of the proposal it looks to be in a stable state.